tests/Fixer/Phpdoc/PhpdocTypesOrderFixerTest.php

```diff
@@ -605,6 +617,9 @@ public function provideFixWithAlphaAlgorithmAndNullAlwaysLastCases()
                 '<?php /** @var Foo|Foo[]|Foo\Bar|Foo_Bar|null */',
                 '<?php /** @var Foo[]|null|Foo|Foo\Bar|Foo_Bar */',
             ],
+            [
+                '<?php /** @return array<int, callable(array<string, null|string> , DateTime): bool> */',
```

Comment:
Hi, I tried to handle the `null|string` inside the `callable()`, but it's too much effort for me right now.
I hope you agree to merge this as a bugfix, and postpone the handling of `callable()` internal types.

Reply:
Merging this PR would improve the situation even if we know that some cases are still not supported so :+1: